gsy_framework/sim_results/kpi.py

```diff
@@ -163,23 +163,24 @@ def _dissipate_infinite_consumption(self, trade: Dict):
             self.total_self_consumption_wh += trade["energy"] * 1000
             self.demanded_buffer_wh += trade["energy"] * 1000
 
-    def _accumulate_energy_trace(self, core_stats):
+    def _accumulate_energy_trace(self, core_stats: Dict, p2p: bool):
         for target_area_uuid in self.areas_to_trace_list:
             target_core_stats = core_stats.get(target_area_uuid, {})
             for trade in target_core_stats.get("trades", []):
                 self._accumulate_self_consumption(trade)
                 self._accumulate_self_production(trade)
                 self._accumulate_self_consumption_buffer(trade)
                 self._dissipate_self_consumption_buffer(trade)
-                self._accumulate_infinite_consumption(trade)
-                self._dissipate_infinite_consumption(trade)
+                if p2p:
+                    self._accumulate_infinite_consumption(trade)
```

Comment:
It would be nice to understand better these _accumulate_infinite_consumption and _dissipate_infinite_consumption. Could you please change descriptions of these functions?

Reply:
as mentioned offline, I will add some renaming of the functions and doc strings in this module.